src/crypto/README.md

```diff
@@ -310,12 +310,12 @@ crypto.randomFill(buf, (err, buf) => {
 For the legacy Node.js crypto API, asynchronous single-call
 operations use the traditional Node.js callback pattern, as
 illustrated in the previous `randomFill()` example. In the
-Web Crypto API (accessible via `require('node:crypto').webcrypto`),
+Web Crypto API (accessible via `globalThis.crypto`),
 all asynchronous single-call operations are Promise-based.
 
 ```js
 // Example Web Crypto API asynchronous single-call operation
-const { subtle } = require('node:crypto').webcrypto;
+const { subtle } = globalThis.crypto;
```

Comment:
I am pretty sure this exact change was proposed before and I was against it at the time (I can't seem to find the PR/comment right now, sorry) because this does not work in the REPL of most stable versions of Node.js. IIRC `globalThis.crypto` is still the same as `require('node:crypto')` in all maintained LTS versions.

Reply:
Actually, in REPL `const { subtle } = globalThis.crypto;` ends up with the exact same assignment in LTS 16, LTS 18 as well as current.

Reply:
Ah, right, I forgot about `require('node:crypto').subtle`. Thanks for the reminder :)